web/tendrl/user_password_edit_dialog.rst

```diff
@@ -0,0 +1,67 @@
+Change user password at user "Edit" page
+*****************************************
+
+:authors: 
+          - mkudlej@redhat.com
+
+Description
+===========
+
+Change user password at user "Edit" page for these user types:
+ 1. *Admin* user
+ 2. *Power User* user
+ 3. *Read-only* user
+
+Design describing "Edit" page `https://redhat.invisionapp.com/share/KNB25OEQT#/screens/226063805`
+
+Setup
+=====
+
+#. Login user into Tendrl, see :doc:`/web/tendrl/login_positive`
+#. Create 3 users mentioned in Description, see :doc:`/web/tendrl/user_create`
+
+Test Steps
+==========
+
+.. test_action::
+   :step:
+       Click on *Admin* -> *Users* in left menu.
```

Comment:
Just a note, that Users could be elsewhere later. Probably not though, because they ignore designs.